src/slo/sli_view_querier.py

```diff
@@ -16,11 +16,3 @@ def query(self):
         logging.info("Executing query: %s", query)
         query_results = self.big_query.execute_query(query)
         return self.query_specification.format_query_results(query_results, self.snapshot_time), self.snapshot_time
-
-    @staticmethod
-    def sli_entry_to_table_reference(table):
```

Comment:
I'm wondering why you moved it from here to both specification duplicating the same code? What was the reason?

Reply:
answered in comment under Przemek's comment